docs/Makefile

```diff
@@ -12,6 +12,12 @@ APIDOCDIR_PREDICTION   = apidoc_prediction
 SPHINXAPIDOC  = sphinx-apidoc
 PROJDIR       = ../
 
+# Internal variables.
+ALLSPHINXOPTS   = -d $(BUILDDIR)/doctrees $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) .
+
+# Internal variables.
+ALLSPHINXOPTS   = -d $(BUILDDIR)/doctrees $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) .
```

Comment:
I guess this happened during merging at some point? :)

Reply:
Yes @WGierke I am having a difficult time making this work 😭